doc/users/index.rst

```diff
@@ -5,12 +5,12 @@ To generate the certificate and private key of trusted users should be generated
 
 .. code-block:: bash
 
-    $ CCF/tests/keygenerator.sh --name user1
+    $ keygenerator.sh --name user1
     -- Generating identity private key and certificate for participant "user1"...
     Identity private key generated at:   user1_privk.pem
     Identity certificate generated at:   user1_cert.pem (to be registered in CCF)
 
-.. note:: See :ref:`developers/cryptography:Algorithms and Curves` for the list of supported cryptographic curves.
+.. note:: The ``keygenerator.sh`` script can be found in the :ref:`CCF Python package <users/python_tutorial:Python Client Tutorial>` or in the CCF repo under ``python/utils``. This is a simple wrapper around openssl - CCF will accept key may generated by other tools, as long as they are in the correct format and on a supported cryptographic curve. See :ref:`developers/cryptography:Algorithms and Curves` for the list of supported cryptographic curves.
```

Comment:
"can be found in" -> "is installed as part of" maybe, since `keygenerator.sh` & co get installed in the path when `pip install ccf` is run?